Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -12,18 +12,87 @@ static void Main(string[] args)
             Test("42*?47=1974", -1);
             Test("2*12?=247", -1);
             Console.ReadKey(true);
+            
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
+        public static int FindDigit(string equation)
+        {
             // Add your code here.
-            throw new NotImplementedException();
+            int l = equation.Length;
+            int m = l+1;
+            int x = 0;
+            char[] ch = equation.ToCharArray();
+            
+             for(int i = 0 ; i < l ; i++){

```

Comment:
any specific reason to use for loop? Can we avoid it? One could directly find index of '?'.